lib/format/formatter.js

```diff
@@ -50,7 +50,7 @@ class Formatter {
       console.log('skipping format, could not find tool', tool) // eslint-disable-line no-console
       return null
     }
-    const options = this.goconfig.executor.getOptions('project')
+    const options = this.goconfig.executor.getOptions('project', editor)
```

Comment:
shouldn't the formatter run in the context of the file it is formatting? So use `file` instead of `project`?

Reply:
If format tool is goimports and the project is using modules then the working dir needs to be project dir in order to resolve module dependencies and suggest the correct imports.